### DISCUSSION TITLE
Install SSM Agent to AWS AMI image

### DISCUSSION THREAD

**Discussion by liesislukas** [meilisearch/product#576] [Feedback & Feature Proposal]
Please install AWS's SSM Agent so it would not require to open SSH port and manage SSH key. Would be able to use Session Manager instead and access management would be inside IAM permissions instead, which is way more secure.

**Comment by oluademola** [comment]
Hello @liesislukas 

This is indeed the current configuration. We will move this to the appropriate repo because it is not a bug but a feature request.

We appreciate your feedback. Thank you very much.

**Comment by gmourier** [comment]
Hello 👋

I'm locking that discussion; this feature request has been transferred to the appropriate repository as an [issue](https://github.com/meilisearch/meilisearch-aws/issues/260).

Thank you!
